app/views/transferSummary.scala.html

```diff
@@ -0,0 +1,92 @@
+@import java.util.UUID
+@import viewsapi.FormFunctions._
+@import helper.{inputRadioGroup => _, _}
+
+@(consignmentId: UUID, transferSummaryForm: Form[TransferSummaryData])(implicit request: RequestHeader, messages: Messages)
+@main(Messages("transferSummary.header")) {
+@defining(play.core.PlayVersion.current) { version =>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <a href="@routes.DashboardController.dashboard()" class="govuk-back-link">Back</a>
+            @progressIndicator(Messages("transferSummary.progress"))
+            <h1 class="govuk-heading-xl">@Messages("transferSummary.title")</h1>
+            <dl class="govuk-summary-list">
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.seriesReference")
+                    </dt>
+                    <dd class="govuk-summary-list__value summary-list__right-column">
+
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.consignmentReference")
+                    </dt>
+                    <dd class="govuk-summary-list__value summary-list__right-column">
+
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.transferringBody")
+                    </dt>
+                    <dd class="govuk-summary-list__value summary-list__right-column">
+
+                    </dd>
+                </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        @Messages("transferSummary.printableReport")
```

Comment:
Could you remove this row too, for the same reason as in [this comment](https://github.com/nationalarchives/tdr-transfer-frontend/pull/226/files#r512674187)?

Reply:
Sure, will do.